modules/blocks.js

```diff
@@ -466,7 +467,7 @@ __private.applyBlock = function (block, broadcast, cb, saveBlock) {
 						err = ['Failed to apply transaction:', transaction.id, '-', err].join(' ');
 						library.logger.error(err);
 						library.logger.error('Transaction', transaction);
-						// TODO: Send a numbered signal to be caught by forever to trigger a rebuild.
+						// Fatal error, memory tables will be inconsistent
```

Comment:
We should give the user a message why it was killed instead of just killing it because of tx failure